lib/pages/portal/service/website_provider.dart

```diff
@@ -103,8 +106,12 @@ class WebsiteProvider with ChangeNotifier {
   }
 
   /// Initializes the number of visits of websites with the value stored from Firebase.
+  ///If no [uid] is provided, store the data locally instead.
```

Comment:
Nit: space after comment
```suggestion
  /// If no [uid] is provided, store the data locally instead.
```